src/test/java/uk/gov/pay/api/json/CreateCardPaymentRequestDeserializerTest.java

```diff
@@ -473,53 +460,22 @@ public void deserialize_shouldThrowValidationException_whenDelayedCaptureIsNumer
     }
 
     @Test
-    public void deserialize_shouldThrowValidationException_whenMotoIsNotABoolean() throws Exception {
+    @Parameters({"null", "\"true\"", "0"})
+    public void deserialize_shouldThrowValidationException_whenMotoIsNotABoolean(String value) throws Exception {
```

Comment:
You'll need to do `(@Nullable String value)` to sure the "null" parameter is actually null and not "null". See https://github.com/alphagov/pay-publicapi/blob/master/src/test/java/uk/gov/pay/api/it/validation/PaymentResourceMetadataValidationFailuresIT.java#L67-L70 for an example.

Reply:
Also the test name could be renamed to just `deserialize_shouldThrowValidationException`

Reply:
Done